apps/api/src/schema/websoc.ts

```diff
@@ -118,30 +137,71 @@ export const websocQuerySchema = z.object({
       }
       return parsedNums;
     }),
-  instructorName: z.string().optional(),
-  days: daysSchema.optional(),
-  building: z.string().optional(),
-  room: z.string().optional(),
+  instructorName: z.string().optional().openapi({
+    description: "The instructor name of the course",
+    example: "Anteater, P.",
+  }),
+  days: daysSchema.optional().openapi({
+    description: "The days of the week for when the course meets",
+    example: "M,Tu,W,Th",
+  }),
+  building: z.string().optional().openapi({
+    description: "The building where the course is held",
+    example: "ALP",
+  }),
+  room: z.string().optional().openapi({
+    description: "The room where the course is held",
+    example: "1100",
+  }),
   division: z
     .enum(courseLevels)
     .or(z.literal("ANY"))
     .optional()
+    .openapi({
+      description:
+        "The type of course level of the course (e.g. Lower Division, Upper Division, or Graduate)",
+      example: "LowerDiv",
+    })
     .transform((x) => (x === "ANY" ? undefined : x)),
   sectionType: z
     .union([z.enum(anyArray), z.enum(websocSectionTypes)])
     .optional()
+    .openapi({
+      description: "The type of section of the course (e.g. Lecture, Discussion, Lab)",
+      example: "Lec",
+    })
     .transform((x) => (x === "ANY" ? undefined : x)),
   fullCourses: z
     .enum(fullCoursesOptions)
     .optional()
+    .openapi({
+      description: "The enrollment status of the course",
+      example: "SkipFull",
+    })
     .transform((x) => (x === "ANY" ? undefined : x)),
-  cancelledCourses: z.enum(cancelledCoursesOptions).optional(),
-  units: z.optional(z.literal("VAR").or(z.string())),
-  startTime: timeSchema.optional(),
-  endTime: timeSchema.optional(),
+  cancelledCourses: z.enum(cancelledCoursesOptions).optional().openapi({
+    description: "Include or exclude cancelled courses",
+    example: "Exclude",
+  }),
+  units: z.optional(z.literal("VAR").or(z.string())).openapi({
+    description: "The number of units for the courses",
+    example: "4",
+  }),
+  startTime: timeSchema.optional().openapi({
+    description: "The course start time",
+    example: "08:00",
+  }),
+  endTime: timeSchema.optional().openapi({
+    description: "The course end time",
+    example: "17:00",
+  }),
   excludeRestrictionCodes: z
     .string()
     .optional()
+    .openapi({
+      description: "Exclude courses with specific restriction codes",
```

Comment:
You may want to state precisely that this list should be comma-separated, since that is the language developers understand. It would also be a nice addition to link to the official registrar's page explaining what these codes mean (though not strictly needed).